Adds Prisma setup fn (#2693)
wasp-lang/wasp: web/docs/data-model/databases.md

You'll often want to call `wasp db seed` right after you run `wasp db reset`, as it makes sense to fill the database with initial data after clearing it.
:::

## Customising the Prisma Client

Wasp interacts with the database using the [Prisma Client](https://www.prisma.io/docs/orm/prisma-client).
To customize the client, define a function in the `app.db.prismaSetupFn` field that returns a Prisma Client instance.
This allows you to configure features like [logging](https://www.prisma.io/docs/orm/prisma-client/observability-and-logging/logging) or [client extensions](https://www.prisma.io/docs/orm/prisma-client/client-extensions):

<Tabs groupId="js-ts">
  <TabItem value="js" label="JavaScript">
    ```wasp title=main.wasp
    app MyApp {
      title: "My app",
      // ...
      db: {

>         prismaSetupFn: import { setupPrisma } from "@src/prisma"

Author: I was trying to stay consistent with all other setupFn APIs, to be honest. In hindsight, having a function gives us the control when to run it. That's one benefit that I can think of right now :)

  Defines the seed functions you can use with the `wasp db seed` command to seed your database with initial data.
  Read the [Seeding section](#seeding-the-database) for more details.

- `prismaSetupFn: ExtImport`

>   Defines a function that sets up the Prisma Client instance. Wasp expects you to return a Prisma Client instance from this function.

Reviewer: ```suggestion
  Defines a function that sets up the Prisma Client instance. Wasp expects it to return a Prisma Client instance.
```
